ner_v2/detectors/temporal/date/en/test.yaml

```diff
@@ -0,0 +1,337 @@
+args:
+  today:
+    dd: 15
+    mm: 2
+    yy: 2019
+tests:
+  en:
+    - message: 17/12/19
+      bot_message: null
+      outputs:
+        - dd: 17
+          mm: 12
+          yy: 2019
+          property: null
+          range: null
+          repeat: null
+    - message: from 20th Dec to 30th Jan
+      bot_message: null
+      outputs:
+        - dd: 20
+          mm: 12
+          yy: 2019
+          property: null
+          range: start
+          repeat: null
+        - dd: 30
+          mm: 1
+          yy: 2020
+          property: null
+          range: end
+          repeat: null
+    - message: from this Monday to next Tuesday
+      bot_message: null
+      outputs:
+        - dd: 18
+          mm: 2
+          yy: 2019
+          property: null
+          range: start
+          repeat: null
+        - dd: 19
+          mm: 2
+          yy: 2019
+          property: null
+          range: end
+          repeat: null
+    - message: wake me up today and tomorrow at 8 am
+      bot_message: null
+      outputs:
+        - dd: 15
+          mm: 2
+          yy: 2019
+          property: null
+          range: null
+          repeat: null
+        - dd: 16
+          mm: 2
+          yy: 2019
+          property: null
+          range: null
+          repeat: null
+    - message: I missed my yesterdays appointment, so please reschedule to day after tomorrow
+      bot_message: null
+      outputs:
+        - dd: 14
+          mm: 2
+          yy: 2019
+          property: null
+          range: null
+          repeat: null
+        - dd: 17
+          mm: 2
+          yy: 2019
+          property: null
+          range: null
+          repeat: null
+    - message: Wake me up on 23rd September 2022
+      bot_message: null
+      outputs:
+        - dd: 23
+          mm: 9
+          yy: 2022
+          property: null
+          range: null
+          repeat: null
+    - message: Wake me up on 23 September 2022
+      bot_message: null
+      outputs:
+        - dd: 23
+          mm: 9
+          yy: 2022
+          property: null
+          range: null
+          repeat: null
+    - message: Wake me up on 23 Sept 2022
+      bot_message: null
+      outputs:
+        - dd: 23
+          mm: 9
+          yy: 2022
+          property: null
+          range: null
+          repeat: null
+    - message: Wake me up on 23rd September
+      bot_message: null
+      outputs:
+        - dd: 23
+          mm: 9
+          yy: 2019
+          property: null
+          range: null
+          repeat: null
+    - message: Wake me up everyday at 8 am
+      bot_message: null
+      outputs:
+        - dd: 15
+          mm: 2
+          yy: 2019
+          property: null
+          range: null
+          repeat:
+            - unit: 15
+            - interval: 1
```

Comment:
Remove `-` as this will be converted to `{"repeat": [{"unit": 15}, {"interval": 1}]}`

Reply:
Shouldn't we give repeat every X days in the next N days from today? Again depends on how users mention repeats
```
repeat:
    every: 1
    in_next: 30
```

Better names that would be easier to understand are welcome

Reply:
Removed 'unit' for now. We can discuss and add later. Also Like you said, I don't think so NEr should assign any defaults here. For now, I have kept only 'interval'. We can add more keys later as we get more clarity. But NER should add that param only if specified exclusively by user.